apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CarrierClaimTransformerV2Test.java

```diff
@@ -1228,6 +1230,89 @@ public void shouldHaveProcedureList() {
     assertEquals(0, eob.getProcedure().size());
   }
 
+  /** Test that should not have a npi entry for organization. */
+  @Test
+  public void shouldNotHaveNpiEntryForOrg() throws IOException {
```

Comment:
Nit: We should fully describe the case we're testing; should not have an npi entry for organization _when the claim's organization npi is empty_; can adjust the name like this:
`shouldNotHaveNpiEntryForOrgWhenNoOrganizationNpi`
or adjust the doc to describe this, or both

Reply:
done